src/lib/djerba/plugins/pwgs/analysis/plugin.py

```diff
@@ -61,11 +61,18 @@ def extract(self, config):
         data = self.get_starting_plugin_data(wrapper, self.PLUGIN_VERSION)
         workspace_dir = self.workspace.get_work_dir()
         # Read from the case_overview JSON file and populate results
-        with open(os.path.join(workspace_dir, "pWGS_case_overview_output.json"), 'r') as json_file:
-            json_data = json.load(json_file)
-            assay = json_data.get("results", {}).get("assay", "Assay name not found")
-            primary_cancer = json_data.get("results", {}).get("primary_cancer", "Primary cancer not found")
-            study_title = json_data.get("results", {}).get("study_title", "Study title not found")
+        json_file_path = os.path.join(workspace_dir, "pWGS_case_overview_output.json")
+        try:
```

Comment:
A `try/except` is not a good way of doing this. Lots of things could cause an `IOError`, some of which are serious problems we'd want to know about!

Instead use an `if/else`, with `if os.path.isfile(json_file_path)` instead of `try` and `else` instead of `except IOError`.